code/datums/diseases/advance/symptoms/radiation.dm

```diff
@@ -1,6 +1,6 @@
 /datum/symptom/radiation
-	name = "Ionising Cellular Emission"
```

Comment:
not a spelling error

Reply:
no, you leave it as-is because it's not a spelling error

Reply:
this game is multicultural and made by many people and the spelling reflects that, changing it for no reason (when the spelling is accurate!) is colonialism. change the title to not be "mass typo fixes" if it's not typo fixes